p4runtime_sh/shell.py

```diff
@@ -2393,7 +2442,7 @@ def main():
     if args.verbose:
         logging.basicConfig(level=logging.DEBUG)
 
-    setup(args.device_id, args.grpc_addr, args.election_id, args.config)
+    setup(args.device_id, args.grpc_addr, args.election_id, args.config, set_global_client=True)
```

Comment:
I feel like `set_global_client` is actually not needed in `setup`:
* when using p4runtime_sh for scripts, there is no need for global client support IMO
* when using p4runtime_sh as a CLI, setting the global client can be done here, in the `main` function